### PR TITLE
Bug 2187 - Verify BrowserID audience on the server

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/fad8b2daa569014/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/e1c43f28eb6202a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/fad8b2daa569014/output.txt

Total script time: 2.06 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e1c43f28eb6202a/output.txt

Total script time: 1.54 mins

Server is running on http://pr992.butyr.org/templates/basic/
Run the Test Suite: http://pr992.butyr.org/test/

File: cornfield/app.js [hunk 1]
```diff
@@ -80,7 +80,9 @@ app.configure( function() {
     .set('view options', {layout: false});
 });
 
-require('express-browserid').plugAll(app);
+require('express-browserid').plugAll(app, {
```
Comment (secretrobotron) [inline cornfield/app.js]: Let's at least pretend like the code style in this file is uniform and adhere to the agreed-upon spacing we have in Butter as well.

**Pull request merged**